src/main/java/io/cryostat/net/OpenShiftAuthManager.java

```diff
@@ -296,6 +309,29 @@ public AuthenticationScheme getScheme() {
         }
     }
 
+    private Future<Boolean> deleteToken() {
+        try (OpenShiftClient client = clientProvider.apply(getServiceAccountToken())) {
+            String serviceAccountAsOAuthClient = this.getServiceAccountName();
+            List<OAuthAccessToken> userOauthAccessTokens =
+                    client.oAuthAccessTokens().list().getItems().stream()
+                            .filter(t -> t.getClientName().equals(serviceAccountAsOAuthClient))
+                            .collect(Collectors.toList());
+
+            Boolean deleted =
+                    Optional.ofNullable(client.oAuthAccessTokens().delete(userOauthAccessTokens))
+                            .orElseThrow(TokenNotFoundException::new);
```

Comment:
Will this delete tokens for every user? Say Alice and Bob are both logged into Cryostat. One of them logs out, would the other be logged out as well?

Reply:
Oops, I think I mistakenly assumed that all tokens in a single namespace correspond to a single user. I'll add a fix to compare usernames too. Do you know of another user besides `kube:admin` that I can use to test this?

Reply:
There's also the `developer` user in a default CRC install. But the username could be basically any arbitrary string, really.

Reply:
> There's also the developer user in a default CRC install

Is there an equivalent user for Quicklab?

> the username could be basically any arbitrary string

Good point, I should compare user UIDs instead.

Reply:
I found this article helpful before: https://medium.com/kubelancer-private-limited/create-users-on-openshift-4-dc5cfdf85661.

You'll need to authorize the new user by adding them to roles with:
`oc adm policy add-role-to-user`
The `edit` cluster role will give you most built-in permissions you need, then there are a variety of cluster roles for our operator that will help for the operator's CRDs: `cryostat-editor-role`, `flightrecorder-editor-role`, and `recording-editor-role`.

Reply:
> then there are a variety of cluster roles for our operator that will help for the operator's CRDs

For future reference, I can add my new user to the same role used to request am OAuth access_token with all of the necessary permissions: 
```
cd ~/cryostat-operator/config/rbac
oc adm policy add-cluster-role-to-user cluster-admin ocadmin --rolebinding-name=oauth_client
```
If I have two cryostat tabs open, with one logged in as `kube:admin` and one as `ocadmin`, various combinations of logging in, refreshing, and logging out do not affect the other tab as expected.